src/core/eme/eme_manager.ts

```diff
@@ -173,90 +183,141 @@ export default function EMEManager(
       } else {
         wantedSessionType = "persistent-license";
       }
-
       return observableConcat(serverCertificate$,
-                              getSession(initializationData, stores, wantedSessionType))
-        .pipe(mergeMap((sessionEvt) =>  {
+                              handleEMESession(wantedSessionType,
```

Comment:
Didn't we want to avoid using "handle" or "manage" in a function name ? As it is not enough evocative of what the function does.

Reply:
Yes, as I said in the PR message, I found this a poor name too, as it can mean anything.

Didn't really know what to do from there.